translators/crate.py

```diff
@@ -2,28 +2,56 @@
 from datetime import datetime, timedelta
 from translators.base_translator import BaseTranslator
 from utils.common import iter_entity_attrs
+import logging
 import statistics
 
+
 # NGSI TYPES: Not properly documented so this might change. Based on experimenting with Orion.
+NGSI_ID = 'id'
+NGSI_TYPE = 'type'
+NGSI_TEXT = 'Text'
+NGSI_DATETIME = 'DateTime'
+
 # CRATE TYPES: https://crate.io/docs/reference/sql/data_types.html
 NGSI_TO_CRATE = {
     "Array": 'array(string)',  # TODO: Support numeric arrays
```

Comment:
do we have a discussion on data types in the documentation pointing to the respective documents?

https://crate.io/docs/crate/reference/en/latest/appendix/data-types.html

Reply:
Good one, we have it in the code, not in the docs. I'll add now.

Reply:
Forget, we do have in the docs. Each type of the conversion is linked to the corresponding crate docs.

Reply:
what about link to NGSI documentation?

Reply:
There's no documentation of ngsi types as far as I know, so the table in the doc lists the types I've seen in Orion docs. I will include such thing as soon as we have it :)